src/DucksNet.API/DTO/EmployeeDTO.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DucksNet.API.DTO;
+
+public class EmployeeDTO
+{
+    public Guid IdSediu { get; set; }
+    public string Surname { get; set; }
```

Comment:
Please fix build warnings. either by creating a constructor or by marking the fields as nullable.